documentation/helpers/ReactLiveBlock/utils.ts

```diff
@@ -0,0 +1,27 @@
+import copy from 'copy-to-clipboard'
+import { useCallback, useEffect, useState } from 'react'
+
+export function useClipboard(value: string) {
```

Comment:
Maybe this hook should be a package in the future ? 🤔

Reply:
Yep :) @andresin87 suggested it also :) I can be moved to hooks packages when we make them.